pkgs/by-name/eu/eudic/package.nix

```
@@ -0,0 +1,104 @@
+{ fetchurl
+, stdenv
+, autoPatchelfHook
+, makeWrapper
+, lib
+, makeDesktopItem
+, copyDesktopItems
+, fontconfig
+# Dependences of applications declaim
+, libnotify
+, libX11
+, libXScrnSaver
+, libXext
+, libXtst
+, libuuid
+, libsecret
+, xdg-utils
+, xdg-utils-cxx
+, at-spi2-atk
+# Dependences of autoPatchelfHook
+, gtk3
+, alsa-lib
+, libdrm
+, e2fsprogs
+, nss
+, libgpg-error
+, libjack2
+, mesa
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eudic";
+  version = "13.5.2";
+
+  src = fetchurl {
+    url = "https://www.eudic.net/download/eudic.deb?v=${finalAttrs.version}";
+    hash = "sha256-UPkDRaqWF/oydH6AMo3t3PUT5VU961EPLcFb5XwOXVs=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  # Depends: libgtk-3-0, libnotify4, libnss3, libxss1, libxtst6, xdg-utils, libatspi2.0-0, libuuid1, libsecret-1-0
+  buildInputs = [
+    libnotify
+    libX11
+    libXScrnSaver
+    libXext
+    libXtst
+    libuuid
+    libsecret
+    xdg-utils
+    xdg-utils-cxx
+    at-spi2-atk
+    # Dependences base on autoPatchelfHook and nix-locate
+    gtk3                # libEGL.so.1 libfreetype.so.6 libfontconfig.so.1
+    alsa-lib            # libasound.so.2
+    e2fsprogs           # libcom_err
+    nss                 # libnss3.so libnssutils3.so
+    libgpg-error        # libgpg-error.so
+    libjack2            # libjack2.so.0
+    mesa                # libgbm.so.1
```

```suggestion
    gtk3
    alsa-lib
    e2fsprogs
    nss
    libgpg-error
    libjack2
    mesa
```
this list can get easily out of date or slightly change over time with package updates. Lets rather not hardcode to many details which then get forgotten when updating the package